<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Suite1" duration-ms="27870" started-at="2021-11-29T11:23:20 IST" finished-at="2021-11-29T11:23:47 IST">
    <groups>
    </groups>
    <test name="Nopackage" duration-ms="27870" started-at="2021-11-29T11:23:20 IST" finished-at="2021-11-29T11:23:47 IST">
      <class name="org.cbsl.testcases.TC_AddEmployee">
        <test-method status="PASS" signature="setReport()[pri:0, instance:org.cbsl.testcases.TC_AddEmployee@18bf3d14]" name="setReport" is-config="true" duration-ms="165" started-at="2021-11-29T11:23:20 IST" finished-at="2021-11-29T11:23:20 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setReport -->
        <test-method status="PASS" signature="launch(java.lang.reflect.Method)[pri:0, instance:org.cbsl.testcases.TC_AddEmployee@18bf3d14]" name="launch" is-config="true" duration-ms="9767" started-at="2021-11-29T11:23:20 IST" finished-at="2021-11-29T11:23:30 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void org.cbsl.testcases.TC_AddEmployee.loginTest()]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launch -->
        <test-method status="FAIL" signature="loginTest()[pri:0, instance:org.cbsl.testcases.TC_AddEmployee@18bf3d14]" name="loginTest" duration-ms="16080" started-at="2021-11-29T11:23:30 IST" finished-at="2021-11-29T11:23:46 IST">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [false] but found [true]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:819)
at org.testng.Assert.assertEquals(Assert.java:829)
at org.cbsl.testcases.TC_AddEmployee.loginTest(TC_AddEmployee.java:36)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:284)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:119)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest -->
        <test-method status="PASS" signature="getResult(org.testng.ITestResult)[pri:0, instance:org.cbsl.testcases.TC_AddEmployee@18bf3d14]" name="getResult" is-config="true" duration-ms="1054" started-at="2021-11-29T11:23:46 IST" finished-at="2021-11-29T11:23:47 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=loginTest status=FAILURE method=TC_AddEmployee.loginTest()[pri:0, instance:org.cbsl.testcases.TC_AddEmployee@18bf3d14] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getResult -->
        <test-method status="PASS" signature="endReport()[pri:0, instance:org.cbsl.testcases.TC_AddEmployee@18bf3d14]" name="endReport" is-config="true" duration-ms="725" started-at="2021-11-29T11:23:47 IST" finished-at="2021-11-29T11:23:47 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- endReport -->
      </class> <!-- org.cbsl.testcases.TC_AddEmployee -->
    </test> <!-- Nopackage -->
  </suite> <!-- Suite1 -->
</testng-results>
